osf/migrations/0148_auto_20181213_2026.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
Nit: For migrations that aren't completely auto-generated, it's good to use a descriptive filename.